package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "retryify",
-  "version": "1.2.1",
+  "version": "0.0.0-semantic-relase",
```

Comment:
what's up with this `-semantic-release` on the version? does that get stripped out somewhere?

Reply:
We could basically put anything there, when semantic-release publishes the package to npm it replaces the version with the correctly computed version before pushing the files to npm. They suggest putting `0.0.0-semantic-release`so that it's clear that the version property should not be modified manually